rest/RESTClient.ts

```diff
@@ -63,11 +70,21 @@ export default class RESTClient {
         const discordBucket = api.headersIn['x-ratelimit-bucket'] as string | undefined;
         if (api.headersIn['x-ratelimit-global'] === 'true') {
           this.globallyRateLimited = true;
+
           setTimeout(() => {
             this.globallyRateLimited = false;
+
+            while (this.queue.length >= 1) {
+              const _request = this.queue.shift();
+
+              // @ts-ignore ts(2722) "Cannot invoke an object which is possibly 'undefined'."
+              _request();
```

Comment:
Can just concatenate this
```suggestion
              this.queue.shift()!();
```